lib/src/analyzer_impl.dart

```diff
@@ -122,8 +125,13 @@ class AnalyzerImpl {
   void prepareErrors() {
     for (Source source in sources) {
       context.computeErrors(source);
+
       var sourceErrors = context.getErrors(source);
       errorInfos.add(sourceErrors);
+
+      if (options.strongMode) {
+        errorInfos.add(strongChecker.computeErrors(source));
+      }
```

Comment:
Sorry if this was clarified elsewhere but if only for my benefit, am I right that if we take this approach we'll potentially see a pile up of potentially contradictory warnings in cases where the strong checker and the the more relaxed one don't agree?